book/src/tutorial/parser.md

```diff
@@ -63,7 +63,7 @@ It's generally better to structure tracked functions as functions of a single sa
 
 ### The `return_ref` annotation
 
-You may have noticed that `parse_statements` is tagged with `#[salsa::tracked(return_ref)]`.
+You may have noticed that `parse_statements` is tagged with `#[salsa::tracked(return_ref)]`. [The function isn't actually tagged with `return_ref`; is the text incorrect or the code example?]
```

Comment:
I'm planning to do some major changes to this section

Reply:
By this I mean the tutorial

Reply:
I have a branch that extends the tutorial to include the type checker etc, and I have to go back and rewrite some of the sections from the parser as a result.

Reply:
Ok, I'll leave this text as-is for now then.